packages/playback-core/src/index.ts

```diff
@@ -337,17 +337,76 @@ export const getLiveEdgeStart = (mediaEl: HTMLMediaElement) => {
   return seekable.end(seekable.length - 1) - liveEdgeStartOffset;
 };
 
-const isApproximatelyEqual = (x: number, y: number, moe = 0.001) => Math.abs(x - y) <= moe;
-const isApproximatelyGTE = (x: number, y: number, moe = 0.001) => x > y || isApproximatelyEqual(x, y, moe);
+const DEFAULT_ENDED_MOE = 0.034;
```

Comment:
What does MOE stand for?

Reply:
"margin of error". MOE is a common acronym for it, but this has come up in the past since not all devs are familiar with the terminology.

Reply:
Ah, that makes sense. Familiar with the term, but had never come across it abbreviated before somehow

Reply:
I just like naming my variables after Simpsons characters. 😄 jk